chatminer/chatparsers.py

```diff
@@ -147,6 +149,8 @@ def _infer_datatime_format(self):
         max_first = 0
         max_second = 0
         for line in self.messages:
+			if "Passcode" in line or "Meeting ID" in line: # TODO: Zoom links cause an error. This is a temporary workaround.
+				continue
```

Comment:
Would you mind copying over an example for a line containing a Zoom link? We should solve this more generic to catch similar issues as well